lib/query-interface.js

```diff
@@ -637,8 +637,8 @@ class QueryInterface {
     );
   }
 
-  increment(instance, tableName, values, identifier, options) {
-    const sql = this.QueryGenerator.incrementQuery(tableName, values, identifier, options.attributes);
+  incrementOrDecrement(instance, tableName, values, identifier, options) {
```

Comment:
We will still keep `increment / decrement` in query generator , which will call `incrementOrDecrementQuery` with different operators